docs/content/integrations/dbt/quickstart.mdx

```diff
@@ -0,0 +1,246 @@
+       .resolve(),
+   )
+   ```
+
+3. In your `assets.py` file, add the following code. Using the `dbt_assets` decorator allows Dagster to create a definition for how to compute a set of dbt resources, described by a `manifest.json`.
+
+   ```python file=/integrations/dbt/quickstart.py startafter=start_dbt_assets_example endbefore=end_dbt_assets_example
+   from dagster import AssetExecutionContext
+   from dagster_dbt import DbtCliResource, dbt_assets
+
+   from .project import my_project
+
+   @dbt_assets(manifest=my_project.manifest_path)
+   def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
+       yield from dbt.cli(["build"], context=context).stream()
+   ```
+
+4. In your `definitions.py` file, update your Definitions object to include the newly created objects.
+
+   ```python file=/integrations/dbt/quickstart.py startafter=start_dbt_definitions_example endbefore=end_dbt_definitions_example
+   from dagster import Definitions
+   from dagster_dbt import DbtCliResource
+
+   from .assets import my_dbt_assets
+   from .project import my_project
+
+   defs = Definitions(
+       ...,
+       assets=[
+           ...,
+           # Add the dbt assets alongside your other asset
+           my_dbt_assets
+       ],
+       resources={
+           ...,
+           # Add the dbt resource alongside your other resources
+           "dbt": DbtCliResource(project_dir=my_project),
+       },
+   )
+   ```
+
+With these changes, your existing Dagster project is ready to run your dbt project.
+
+</TabItem>
+</TabGroup>
+
+---
+
+## Run your dbt project in Dagster's UI
+
+Now that your code is ready, you can run Dagster's UI to take a look at your dbt project.
+
+1. Change directories to the Dagster project directory:
+
+   ```shell
+   cd my_dagster_project/
+   ```
+
+2. To start Dagster's UI, run the following:
+
+   ```shell
+   dagster dev
+   ```
+
+   Which will result in output similar to:
+
+   ```shell
+   Serving dagster-webserver on http://127.0.0.1:3000 in process 70635
+   ```
+
+3. In your browser, navigate to <http://127.0.0.1:3000>. The page will display the asset graph for the job created by the schedule definition:
+
+  <!--
+  ![Asset graph for your job in Dagster's UI, containing dbt models loaded as Dagster assets](/images/integrations/dbt/quickstart/asset_graph.png)
+  -->
+
+<Image
+alt="Asset graph for your job in Dagster's UI, containing dbt models loaded as Dagster assets"
+src="/images/integrations/dbt/quickstart/asset_graph.png"
+width={2688}
+height={1680}
+/>
+
+3. In Dagster, running a dbt model corresponds to _materializing_ an asset. Because a schedule definition has been added to your code location, your assets will be materialized at the next cron tick. The assets can also be materialized manually by click the **Materialize all** button near the top right corner of the page. This will launch a run to materialize the assets.
+
+---
+
+## Using dbt with Dagster in a single file
```

Comment:
This section should just be the default option in "Load your dbt project in Dagster".

Reply:
That makes sense. I also updated the `Run your dbt project in Dagster's` UI to handle both file and directory use cases. Done in [0211d97](https://github.com/dagster-io/dagster/pull/21240/commits/0211d9715ebdaeeb56ce71b4f984eb3b3b370527)